core/server/master/src/main/java/alluxio/master/file/meta/InodeView.java

```diff
@@ -103,12 +103,21 @@
   boolean isDeleted();
 
   /**
-   * @return true if the inode is a directory, false otherwise
+   * Checks whether the inode is a directory or not.
```

Comment:
`Checks whether the inode is a directory returning a boolean value indicating if it's or not`. What do you think?